UnicornRender/video/include/unicorn/video/vulkan/Renderer.hpp

```diff
@@ -57,8 +57,8 @@ struct SwapChainSupportDetails
 /** @brief Camera data */
 struct UniformCameraData
 {
-    glm::mat4 view;
-    glm::mat4 proj;
+    glm::mat4 view = glm::mat4();
```

Comment:
It does not look like this struct and 
```cpp
struct Camera
{
    glm::mat4 projection = glm::mat4();
    glm::mat4 view = glm::mat4();
}
```
has any difference as of now.

Reply:
Thats just data for shader passing. But for consistent i will swap this two

Reply:
Well, will you?

Reply:
Also why is naming different? `proj` vs `projection`?

Reply:
Order is still different from UnicornRender/video/include/unicorn/video/Camera.hpp